pkg/settings/parser/parser.go

```diff
@@ -610,6 +610,16 @@ func (a *Arguments) parseLongOption(arg, param string) (usedNext bool, err error
 }
 
 func (a *Arguments) parseStdin() error {
+	fi, err := os.Stdin.Stat()
+	if err != nil {
+		return err
+	}
+
+	// Ensure data is piped
+	if (fi.Mode() & os.ModeCharDevice) != 0 {
+		return errors.New(gotext.Get("argument '-' specified without input on stdin"))
```

Comment:
nit: using the dynamic errors like this doesn't allow `errors.Is` to be used against it

(does not require fix as we probably don't need to catch this)

Reply:
What should be used instead?

Reply:
Normally package variables are used in this case (https://stackoverflow.com/a/62292589) and error structs when the message has variables